storage/container_storage_interface/persistent-storage-csi-ebs.adoc

```diff
@@ -0,0 +1,30 @@
+[id="persistent-storage-csi-ebs"]
+= AWS Elastic Block Store CSI Operator
+include::modules/common-attributes.adoc[]
+:context: persistent-storage-csi-ebs
+
+toc::[]
+
+== Overview
+
+Storage vendors have traditionally provided storage drivers as part of Kubernetes. With the implementation of the Container Storage Interface (CSI), third-party providers can instead deliver storage plug-ins using a standard interface without ever having to change the core Kubernetes code.
```

Comment:
We typically use `drivers` for CSI, and `plugins` for in-tree. Not sure if this is relevant, though...